### PR TITLE
Hicolormos tests

### PR DESCRIPTION
**Pull request by Kelvinrr**
## Description

Adds test for hicolormos 

## Motivation and Context

913Mb

## How Has This Been Tested?

tests pass locally 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [ ] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by acpaquette** [comment]
The MroCtxCube in Fixtures.h still needs to be updated to MroHiriseCube. Also don't we still need the MroCtxCube?

**Comment by Kelvinrr** [comment]
@acpaquette It's more that it was actually a Hirise cube all along and I renamed it wrong in my last PR so this is me correcting it. For some reason when I think MRO my brain immediate defaults to believing CTX being the only instrument for whatever reason.

**Review by kaitlyndlee — DISMISSED** [review]

File: isis/src/mro/apps/hicolormos/hicolormos.cpp [hunk 1]
```diff
@@ -0,0 +1,281 @@
+#include "Application.h"
```
Comment (acpaquette) [inline isis/src/mro/apps/hicolormos/hicolormos.cpp]: Shouldn't need this Application include

**Review by acpaquette — COMMENTED** [review]
One small thing but other than that this looks good to go! :+1:

**Review by acpaquette — DISMISSED** [review]
Just need to fix up the merge conflict. :+1:

**Comment by acpaquette** [comment]
Mmmmm, maybe not? Looks like there are some missing/renamed files in the fixtures?
`1795: C++ exception with description "**I/O ERROR** Unable to open [data/mroKernels/mroHiriseProj.cub]." thrown in SetUp().`

This is just one example but there are a bunch of failing tests that use the `MroHiriseCube`

**Comment by Kelvinrr** [comment]
ugh, things are building fine locally, I'll look into this some more.

**Comment by Kelvinrr** [comment]
Apparently I never pushed cubes up before since I needed to do a git add -f since *.cub is in .gitignore.

**Comment by Kelvinrr** [comment]
I managed to get the Cube as small as it can get it seems, which is still pretty small, so I think this is good for re-review.

**Review by acpaquette — APPROVED** [review]
:+1:

**Pull request merged**